content/en/users/workload-manager/_index.md

```diff
@@ -108,7 +108,7 @@ There are several options to access the service:
 1. Individual researchers who want to do some number crunching for a limited
    period of time, with a reasonable (not too high) number of CPUs > can use
    the catch-all VO resource pool (`vo.access.egi.eu`). Submit a request through
-   the EGI Marketplace selecting:
+   the EGI Marketplace selecting:  
```

Comment:
Here it would be better to have somethign like using `\` at the end of the line instead of the two spaces, it will be more visible and less subject to be removed by error